macros/edr/tests/test_utils/create_elementary_test_table.sql

```diff
@@ -1,6 +1,12 @@
 {% macro create_elementary_test_table(database_name, schema_name, test_name, table_type, sql_query) %}
     {% if execute %}
         {% set temp_table_name = elementary.table_name_with_suffix(test_name, "__" ~ table_type ~ elementary.get_timestamped_table_suffix()).replace("*", "") %}
+        
+        {% set default_identifier_quoting = api.Relation.get_default_quote_policy().get_part("identifier") %}        
+        {% if not adapter.config.quoting.get("identifier",default_identifier_quoting) %}
```

Comment:
@trevdoz  little nitpick:
```suggestion
        {% if not adapter.config.quoting.get("identifier", default_identifier_quoting) %}
```
other then that look great, ill test it and then it can be merged 👍